nilearn/surface/surface.py

```diff
@@ -1273,6 +1273,22 @@ def to_filename(self, filename):
         _data_to_gifti(data, filename)
 
 
+def surface_at_least_2d(input):
```

Comment:
for this one:
- should it be public: may be useful for users
- tempted to call this one just `at_least_2d` in reference to numpy or 1d_to_2d in reference to nibabel 4d_to_3d